### PR TITLE
Almost ready for PyPI

### PR DESCRIPTION
**Pull request by arielah**
I have run this on TestPyPI and it looks okay (https://test.pypi.org/project/wenda-gpu/0.7.3/), I'm almost ready to pull the trigger on publishing it to PyPI. If you have any tips on getting my ducks more in a row, I'd appreciate that.

### CODE REVIEW

**Comment by ben-heil** [comment]
Looks good to me, though I've never actually put together a PyPI package before. I'm not sure who would have more experience though, most of the people I'd refer you to have graduated or moved on. Maybe Milton would know more if you want a more in-depth review?

File: README.md [hunk 1]
```diff
@@ -1,27 +1,82 @@
 # wenda_gpu: fast domain adaptation for genomic data
 
+## Overview
+
 Domain adaptation allows for development of predictive models even in cases with limited or unlabeled sample data, by developing predictors for the data of interest (target data) using labeled data from a similar distribution (source data).
 This repo is a fast implementation of one domain adaptation method, weighted elastic net domain adaptation, or wenda.
```
Comment (ajlee21) [inline README.md:6]: Could link to the wenda paper or github here

File: example/wenda_gpu_quick_usage.ipynb [hunk 2]
```diff
@@ -0,0 +1,505 @@
+{
```
Comment (ajlee21) [inline example/wenda_gpu_quick_usage.ipynb:1]: I thought this notebook was very nicely documented!

File: README.rst [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+
+By default, wenda_gpu creates the following structure in your working
+directory:
+
+::
+
+   working_directory
+       ├── data
+       │   └── prefix
+       │       ├── source_data.tsv
```
Comment (ajlee21) [inline README.rst:61]: Is the source_data.tsv and target_data.tsv something that wenda creates or an input that the user needs to create? Based on the instructions, I'd assume that all these files are created by wenda

File: README.md (hunk 1 above)
Comment (ajlee21) [reply README.md:6]: Oh I see you have this at the bottom so maybe its not needed here. Up to you

**Review by ajlee21 — APPROVED** [review]
Looks great! You'r first pypi package, so exciting!

Sorry if the directions were confusing. I think the main parts are in the `Publish Your Package to PyPI` section of [here](https://realpython.com/pypi-publish-python-package/)

File: README.rst (hunk 3 above)
Comment (arielah) [reply README.rst:61]: Good catch. Those ones a user will have to create. I'll rewrite the section.

**Pull request merged**